Library/Formula/xsd.rb

```diff
@@ -0,0 +1,49 @@
+class Xsd < Formula
+  desc "XML Data Binding for C++"
+  homepage "http://www.codesynthesis.com/products/xsd/"
+  url "http://www.codesynthesis.com/download/xsd/4.0/macosx/i686/xsd-4.0.0-i686-macosx.tar.bz2"
```

Comment:
I know I can install this for myself without putting it into the Homebrew repository, but the whole point of contributing formulas is to make life easier for others. That's why I am suggesting that you push this formula to homebrew/binary if that's the right place for it.